cirq/google/engine/engine_test.py

```diff
@@ -180,11 +180,9 @@
 
 
 def test_repr():
-    v = cirq.google.JobConfig(project_id='my-project-id',
-                              program_id='my-program-id',
-                              job_id='my-job-id')
+    v = cirq.google.JobConfig(program_id='my-program-id', job_id='my-job-id')
 
-    assert repr(v) == ("JobConfig(project_id='my-project-id', "
+    assert repr(v) == ("JobConfig("
```

Comment:
Use `cirq.testing.assert_equivalent_repr(v)` instead of asserting the specific details. You likely need to change the repr to say `cirq.google.JobConfig`.

Reply:
It turns out that this tests for equality of the object as evaluated from the repr, so I also had to provide equality on the JobConfig class as well. Done.